tools/oxide-bar/src/main.rs

```diff
@@ -20,6 +21,17 @@ use oxideipc::state::OxideState;
 
 use crate::config::Config;
 
+#[cfg(debug_assertions)]
```

Comment:
great , thank you. Would it be much of a hassle to also add these log messages to syslog?

Reply:
If there is alot of code duplication in the logging area we could simply make this its own workspace.

Reply:
No not to much hassle, Yes :)
I was thinking to name that workspace common ?:) However, I'm not to sure of the placement, since i don't think it is a tool neither do i like to put it on the level of the src folder...
Maybe you have an idea ?:)

Reply:
well we could rename `tools` to `crates` which is a rusty name and makes sense. Or `workspaces` then its not that descriptive and everything can be placed there.

Reply:
i would propose a seperate ticket for this issue ?? :)

Reply:
https://github.com/DHBW-FN/OxideWM/issues/173

Reply:
great